README.md

```diff
@@ -79,7 +79,7 @@ And also a child folder within the root:
 
 ***Note:*** calls to <tt>getFolder()</tt> will throw an exception if the folder doesn't exist or is not readable.
 
-#Legal
+# Legal
```

Comment:
Since we're already changing this file might as well change the below copyright line to `2013-2019`